tests/virtualization/universal/waitfor_guests.pm

```diff
@@ -77,6 +77,9 @@ sub add_pci_bridge {
         # Apply xml settings to VM. Note: They will be applied after reboot.
         assert_script_run("virt-xml-validate $xml");
         assert_script_run("virsh define $xml");
+
+    } elsif (script_run("grep machine '$xml' | grep 'xenpv'") == 0) {
```

Comment:
@pdostal I've added this to prevent softfailures likes [this one](http://openqa.qam.suse.cz/tests/18824#step/waitfor_guests/351)

Reply:
```suggestion
    } elsif (is_xen_host() && is_pv_guest($guest)) {
```

Reply:
Included, thanks :+1: